body_to_camera: {
  qx: 0.7340130817138820,
  qy: 0.6786860339144344,
  qz: -0.0222104013738924 ,
  qw: -0.0108056148920726,
  tx: 0.0778793685202638,
  ty: -0.0186633438426013,
  tz: -0.0299680661639963}
